android/modules/ui/AndroidManifest.xml

```diff
@@ -4,7 +4,7 @@
 	android:versionCode="1"
 	android:versionName="1.0">
 
-	<uses-sdk android:minSdkVersion="8" android:targetSdkVersion="11" />
+	<uses-sdk android:minSdkVersion="11" android:targetSdkVersion="21" />
```

Comment:
So can we make that specific functionality dependent on API > 10? So all we would lose is the postlayout event for these widgets for API10, which few people probably need in any case? It's better than dropping API 10 altogether, in my opinion.Thanks!